app/src/main/java/net/gsantner/markor/format/markdown/MarkdownHighlighter.java

```diff
@@ -36,7 +35,7 @@
     private static final int MD_COLOR_QUOTE = 0xff88b04c;
     private static final int MD_COLOR_CODEBLOCK = 0xff8c8c8c;
 
-    public MarkdownHighlighter(HighlightingEditor hlEditor, Document document) {
+    public MarkdownHighlighter(HighlightingEditor hlEditor, Document document, MarkdownTextActions actions) {
```

Comment:
Passing MarkdownTextActions in feels inelegant. What do you think @gsantner.

I may remove this functionality.

Reply:
Solved.